src/probabilities_estimators/symbolic/dispersion/spatial_dispersion.jl

```diff
@@ -0,0 +1,132 @@
+using Statistics
+export SpatialDispersion
+import Base.maximum
+
+"""
+    SpatialDispersion <: ProbabilitiesEstimator
```

Comment:
No, this is the documentation string. The information exposed to the user should be `<: ProbabilitiesEstimator` because that's what the user cares about.

Reply:
> This should be a SpatialProbEst{D, P}, right?

As detailed in my first comment to this PR, `SpatialProbEst{D, P}` type is an *implementation detail* introduced only to simplify the codebase, because we now have multiple spatial estimators. This internal structure of the code might change without it ever affecting the end user. 

Without it being stated anywhere in the code or docs for Entropies.jl (woops, maybe be should :D), we follow Julia's version of [https://semver.org/](semantic versioning). Point 1 in semver says that a package must declare a *public API*. The public API is *strictly* what exists in the documentation.

We don't want implementation details to unnecessarily clutter the public API, so it is best to only expose what is needed. `SpatialDispersion` *is* a `SpatialProbEst{D, P}`, but as @Datseris said, that is not relevant to the user.  Documenting `SpatialDispersion <: ProbabilitiesEstimator` is sufficient. Hypothetically, in the future, we might even end up removing `SpatialProbEst` completely. As long as it's not documented, internal changes will have zero effects on the end user.

Sidenote: the name `SpatialDispersion` itself implies that this is a spatial estimator too, so this information is already there implicitly.